lib/elixir/lib/uri.ex

```diff
@@ -73,6 +73,9 @@ defmodule URI do
   Keys and values can be any term that implements the `String.Chars`
   protocol, except lists which are explicitly forbidden.
 
+  To encode a list use keyword list with multiple mentions of a key
```

Comment:
I don't necessarily agree with this advice. It is one of multiple ways of encoding a keyword list. I don't necessarily believe we should promote it above other approaches?